libs/scully/src/lib/startBackgroundServer.ts

```diff
@@ -46,11 +49,11 @@ export function startBackgroundServer(scullyConfig: ScullyConfig) {
     }
   ).on('close', (code) => {
     if (+code > 0) {
-      const message = 'Problem starting background servers ' + code;
+      const message = 'Problem starting background server ' + code;
```

Comment:
There are actually 2 to 3 servers started. not just 1
1. the server that serves the unmodified dist files
2. the server that serves the Scully results, we don't use that, but it makes evaluation of the work done easier
3. optional test data server